### PR TITLE
Add nb-culler cronjob

### PR DESCRIPTION
**Pull request by DanNiESh**
relevant issues: https://github.com/nerc-project/operations/issues/404,
https://github.com/nerc-project/operations/issues/387,
https://github.com/nerc-project/operations/issues/337

In this PR, I didn't change the namespace from our testing namespace `ope-rhods-testing-1fef2f` to `rhods-notebooks` because we need to confirm that everything operates as expected before we proceed to apply this cronjob to rhods-notebooks.

### CODE REVIEW

File: cronjobs/nb-culler/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+                  # Loop through each notebook
+                  while read -r nb ns ts user image size; do
+                      if [[ " $group_members " =~ " $user " ]]; then
+                          echo "$user is in the $group_name group."
+                          cutoff_time=21600
+                          current_time=$(date -u +%s)
+                          timestamp=$(date -d $ts +%s)
+                          difference=$((current_time - timestamp))
+                          if [[ $image != *"ucsls-nerc-rhoai"* ]]; then
```
Comment (dystewart) [inline cronjobs/nb-culler/cronjob.yaml:55]: Let's maybe make this configurable via variable (The image we're checking against) as well? Same reason as  suggestion above: [https://github.com/nerc-project/operations/issues/404](https://github.com/nerc-project/operations/issues/404)

**Review by dystewart — CHANGES_REQUESTED** [review]

**Comment by computate** [comment]
Thanks for working hard on this @DanNiESh and @dystewart to ensure this culler cronjob doesn't affect every notebook. I'm following up with Professor Lance Galletti to identify the uptime constraints for his class as well.

File: cronjobs/nb-culler/cronjob.yaml (hunk 1 above)
Comment (DanNiESh) [reply cronjobs/nb-culler/cronjob.yaml]: I've set these 3 variables to ENV variables and tested them. It's working now.

**Review by computate — APPROVED** [review]
Well done!

**Pull request merged**